### PR TITLE
Fixes for tests that depend on fast statsz

### PR DESCRIPTION
**Pull request by derekcollison**
Also minor fix for new test to cleanup cluster when done.

Signed-off-by: Derek Collison <derek@nats.io>

### CODE REVIEW

File: server/events.go [hunk 1]
```diff
@@ -98,7 +98,11 @@ const (
 // FIXME(dlc) - make configurable.
 var eventsHBInterval = 30 * time.Second
 
-var statszRateLimit = 1 * time.Second
+// Default minimum wate time for sending statsz
```
Comment (bruth) [inline server/events.go:101]:
```suggestion
// Default minimum wait time for sending statsz
```

**Review by bruth — APPROVED** [review]
LGTM

File: server/events.go (hunk 1 above)
Comment (derekcollison) [reply server/events.go:101]: Thanks will fix..

**Comment by derekcollison** [comment]
Thanks..

**Review by wallyqs — APPROVED** [review]

**Pull request merged**